src/blueprints/submit.njk

```diff
@@ -0,0 +1,67 @@
+---
+layout: default
+title: Submit Your Blueprint
+description: <p>Share your Blueprints to help the FlowFuse community build a best-in-class library of commercially useful content and build recognition of your company as top Node-RED experts.</p><p>We are accepting submissions of Blueprints that are commercially valuable and professionally well-constructed. Submissions will be reviewed by the FlowFuse team. Accepted submissions will be featured on the Blueprints page with acknowledgement of the company that authored the Blueprint.</p>
+sitemapPriority: 0.7
+pagination:
+  data: collections.blueprints
+  size: 3
+  alias: blueprints
+  reverse: true
+meta:
+    description: Submit your own Blueprints for publishing in the FlowFuse Blueprint Library
+hubspot:
+    script: "hubspot/hs-form.njk"
+    formId: c627fbcb-a3e0-46dd-978b-122461b7835c
+    cta: "blueprint-upload"
+    reference: "blueprint-upload"
+---
+
+{% extends 'layouts/catalog.njk' %}
+
+{% block title %}
+{{ title }}
+{% endblock %}
+
+{% block description %}
+Share your Blueprints to help the FlowFuse community build a best-in-class library of commercially useful content and build recognition of your company as top Node-RED experts.
+{% endblock %}
+
+{% block actions %}
+<a href="/blueprints" class="ff-btn ff-btn--primary flex-col uppercase">View All Blueprints</a>
+{% endblock %}
+
+{% block content %}
+<div class="text-left max-w-full">
+  <div class="container m-auto text-left md:max-w-6xl px-4">
+    <div class="grid gap-4 pt-12 md:grid-cols-2">
+        <div>
+            <p>We are accepting submissions of Blueprints that are <b>commercially valuable and professionally well-constructed</b>.</p> <p>Submissions will be reviewed by the FlowFuse team.</p>
+            <p>Accepted submissions will be featured on the Blueprints page with acknowledgement of the company that authored the Blueprint.</p>
+        </div>
+        <!-- Form / Calendar Script -->
+        <div class="w-full">
+        {% include hubspot.script %}
+        </div>
+  </div>
+  <div class="w-full py-10 text-center ">
+    <div class="container m-auto max-w-lg md:max-w-4xl">
+        <h2>Existing Blueprints</h2>
+        <p>Here are a few examples of Blueprints from our collection.You can take a look at the full collection in our <a href="/blueprints">Blueprint Library</a></p>
+        <ul class="flex flex-wrap">
+            {%- asyncEach item in blueprints -%}
+            <li class="w-full md:w-1/3 my-2 px-3 pb-6 border-b mt-6 flex flex-col justify-between group">
+                <a href="{{ item.url }}" class="relative w-full flex flex-col hover:no-underline">
+                    <div class="ff-blog-tile overflow-hidden">
+                        <div class="w-full h-auto transition-transform group-hover:scale-105">
```

Comment:
On hover, the corners become sharp again, I've created some classes that I've been using to address this
```suggestion
                        <div class="w-full h-auto transition-transform ff-image-cover ff-image-rounded scale">
```

<img width="1012" alt="Screenshot 2024-10-23 at 12 05 55" src="https://github.com/user-attachments/assets/0bcee20e-31f1-4837-a6a5-9454c1a5143d">